src/aosm/azext_aosm/common/constants.py

```diff
@@ -11,7 +11,7 @@
 VNF = "vnf"
 CNF = "cnf"
 NSD = "nsd"
-
+VNF_NEXUS = "vnfnexus"
```

Comment:
I think adding a dash would make this more readable. i.e., `vnf-nexus` , as long as that doesn't break things on the CLI itself.